content/docs/reference/identity-provider-settings.mdx

```diff
@@ -195,7 +209,17 @@ IDP_PROVIDER=github
 
 **Identity Provider Request Params** lists the parameters you want to include as part of a sign-in request using the OAuth 2.0 code flow.
 
-Downstream application headers will be overwritten by Pomerium's headers on conflict.
+### Defaults
```

Comment:
Not a huge deal, but could we move the **Defaults** section below **How to configure**? Just because that's the order we follow for the rest of these settings.

Reply:
Thanks, fixed. I thought I was following the pattern from the other sections, but clearly I did not. :/